lbz/lambdas/broker.py

```diff
@@ -16,20 +16,19 @@ def __init__(
         event: dict,
         context: object = None,
```

Comment:
According to [#989723417](https://github.com/pdyba/lambdalizator/pull/71#discussion_r989723417), `context` cannot be optional.